requirements.txt

```diff
@@ -1,7 +1,7 @@
 Click==7.0
 requests==2.21.0
 pyyaml>=git+git://github.com/yaml/pyyaml@releases/tag/5.1b3
-devo-sdk==3.0.2
+devo-sdk==3.2.0
```

Comment:
3.2.1

Reply:
This merge will be a major, There are a task pending in the SDK, I'll wait to have the new version of the SDK and update to the new one